src/main/java/com/parasoft/demoapp/service/ItemInventoryMQService.java

```diff
@@ -1,19 +1,38 @@
 package com.parasoft.demoapp.service;
 
 import com.parasoft.demoapp.config.activemq.ActiveMQConfig;
+import com.parasoft.demoapp.dto.InventoryInfoDTO;
+import com.parasoft.demoapp.dto.InventoryOperation;
+import com.parasoft.demoapp.dto.InventoryOperationRequestMessageDTO;
 import com.parasoft.demoapp.dto.InventoryOperationResultMessageDTO;
+import com.parasoft.demoapp.model.industry.OrderItemEntity;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.jms.core.JmsMessagingTemplate;
 import org.springframework.stereotype.Service;
 
 import javax.jms.Destination;
+import java.util.List;
 
 @Service
 public class ItemInventoryMQService {
 
     @Autowired
     private JmsMessagingTemplate jmsMessagingTemplate;
 
+    public void sendToRequestQueue(InventoryOperation operation, String orderNUmber, List<OrderItemEntity> orderItems) {
```

Comment:
I think these methods should be moved to OrderMQService. 
There may be duplication here after this comment is resolved: https://github.com/parasoft/parasoft-demo-app/pull/74/files#r1006198174. Please work with Renee.